grpc/src/main/java/io/stargate/grpc/service/ExceptionHandler.java

```diff
@@ -61,8 +61,10 @@ public void handleException(Throwable throwable) {
       handleException(throwable.getCause());
     } else if (throwable instanceof StatusException
         || throwable instanceof StatusRuntimeException) {
+      LOG.error(getErrorMessage(null, throwable, true));
```

Comment:
but wait the error is still not given as a second parameter to the `LOG.error()`